apps/docsite/project.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "docsite",
+  "$schema": "../../node_modules/nx/schemas/project-schema.json",
+  "sourceRoot": "apps/docsite/src",
+  "projectType": "application",
+  "tags": [],
+  "targets": {
+    "lint": {
+      "executor": "@nx/eslint:lint"
+    },
+    "storybook": {
+      "executor": "@nx/storybook:storybook",
+      "options": {
+        "port": 4400,
```

Comment:
should it start on the same port as storybook for stories? wont it be better to have different port for the docsite?

Reply:
I'd rather go with the default setup provided by NX instead of using a custom value.

Reply:
yup lets use default, it doesn't matter what project type we have, consistency wins